_includes/includes/ui/keyboard-details.php

```diff
@@ -478,44 +332,36 @@ protected static function WriteDownloadBoxes() {
       global $embed, $session_query_q, $embed_win, $embed_version, $pageDevice;
       global $embed_target;
 
-      $deviceboxfuncs = array(
-        "Windows" => "self::WriteWindowsBoxes",
-        "mac" => "self::WritemacOSBoxes",
-        "Linux" => "self::WriteLinuxBoxes",
-        "iPhone" => "self::WriteiPhoneBoxes",
-        "iPad" => "self::WriteiPadBoxes",
-        "Android" => "self::WriteAndroidBoxes"
-      );
+      // We'll write all the different platforms here and then let Bowser determine
+      // which box to show. This is true for both embedded and web-based viewing.
```

Comment:
I assume this is related to the previous comment (about where `$embed` is used regarding download boxes), but I'm still having trouble figuring out exactly where the connection point actually is.

Reply:
> connection point

I'm not sure what you mean?

Reply:
How is `$embed` used by/forwarded to Bowser?

Reply:
`$embed` is not used by Bowser. Bowser sniffs the user-agent to determine the OS, browser, etc. We take the result from Bowser and set the `data-platform` attribute of the `<html>` element to activate the appropriate CSS rules.

Reply:
... so users would see the same thing in iOS via in-app keyboard search as they would via web-surfing in Safari and searching outside the Keyman app?